app/presenters/collection_presenter.rb

```diff
@@ -79,4 +79,10 @@ def created_by
   def created_datetime
     collection.created_at.localtime.strftime('%b %d, %Y, %l:%M%p %Z')
   end
+
+  def editable?
+    return false if collection.depositors.include? Current.user
```

Comment:
This should be an ActionPolicy allowed_to? instead of duplicating auth logic here.